lib/get-babel-options.js

```diff
@@ -26,6 +26,9 @@ function getBabelOptions(addonContext) {
     options.blacklist.push('es6.modules');
   }
 
+  // All browsers which support service workers support async/await
+  options.blacklist.push('regenerator');
```

Comment:
Can you test async await in a service worker registration file on an unsupported browser? See if it doesn't crash and burn on syntax errors?